lib/src/components/Slider/docs/examples/border-selector.tsx

```diff
@@ -0,0 +1,33 @@
+import type { RangeType } from '@/components/Slider'
+import { Slider } from '@/components/Slider'
+
+const Example = () => {
+  const handleChange = (value: number) => {
+    alert(value)
+  }
+
+  const handleChangeRange = (value: RangeType) => {
+    alert(`${value.min} to ${value.max}`)
+  }
+
+  return (
+    <>
+      <Slider
+        borderSelectorColor="secondary-orange"
+        max={100}
+        min={0}
+        onChange={handleChange}
+        value={50}
+      />
+      <Slider.Range
+        borderSelectorColor="secondary-orange"
+        max={100}
+        min={0}
+        onChange={handleChangeRange}
+        value={{ max: 75, min: 25 }}
+      />
+    </>
+  )
+}
+
+export default Example
```

Comment:
Not sure this example is still relevant? As you marked the borderSelectorColor as deprecated?